extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -691,9 +706,29 @@ export class AmpStoryPage extends AMP.BaseElement {
       if (this.isBotUserAgent_) {
         mediaEl.muted = false;
         mediaEl.removeAttribute('muted');
+        if (mediaEl.tagName === 'AUDIO' && mediaEl.paused) {
+          mediaEl.play();
+        }
       } else {
-        return mediaPool.unmute(
-            /** @type {!./media-pool.DomElementDef} */ (mediaEl));
+        mediaEl = /** @type {!./media-pool.DomElementDef} */ (mediaEl);
+        const promises = [mediaPool.unmute(mediaEl)];
+
+        // Audio element might not be playing if the page navigation did not
+        // happen after a user intent, and the media element was not "blessed".
+        // On unmute, make sure this audio element is playing, at the expected
+        // currentTime.
+        if (mediaEl[PLAY_AUDIO_ELEMENT_FROM_TIMESTAMP_PROPERTY_NAME]) {
```

Comment:
Do we need to clear this property so that when we go forward and then back, it doesn't try to do this again?

Reply:
Good catch, thanks! Done. I actually moved it from the element level to the page level, since audio elements could be re-used by the media pool.